qa/smoke-test-http/src/test/java/org/elasticsearch/http/snapshots/RestGetSnapshotsIT.java

```diff
@@ -46,10 +47,16 @@
 // TODO: dry up duplication across this suite and org.elasticsearch.snapshots.GetSnapshotsIT more
 public class RestGetSnapshotsIT extends AbstractSnapshotRestTestCase {
 
+    private static final Settings LARGE_SNAPSHOT_POOL_SETTINGS = Settings.builder()
+        .put("thread_pool.snapshot.core", 5)
+        .put("thread_pool.snapshot.max", 5)
```

Comment:
Can you add a comment on why this need to be raised?

Also, I think 2 would be enough, 1 to be blocked for `test-index-2` and one to ensure `test-index-1` completes?

Reply:
I think you missed this comment?

Reply:
Sorry, I indeed missed it! I've moved `LARGE_SNAPSHOT_POOL_SETTINGS` to `RestGetSnapshotsIT`. I believe 2 is insufficient, the test hangs with 2 threads, but works with 3.